events/wc_titanforged_events.txt

```diff
@@ -1 +1,198 @@
+		trigger = {
+			character = 400502
+		}
+		text = EVTDESC_WCUTF.1_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+	
+
+	trigger = {
+		OR = {
+			character = 400501
+			character = 400502
+		}
+		is_save_game = no
+	}
+
+	immediate = {
+		c_400502 = {
+			save_event_target_as = target_thorim
+		}
+	}
+	option = {
+		name = EVTOPTA_WCUTF.1
+	}
+	option = {
+		name = EVTOPTB_WCUTF.1
+		character_event = { id = WCUTF.2 days = 7 }
+		
+		ai_chance = {
+			factor = 0
+		}
+	}
+	option = {
+		name = EVTOPTC_WCUTF.1
+		character_event = { id = WCUTF.3 days = 7 }
+		
+		ai_chance = {
+			factor = 0
+		}
+	}
+}
+character_event = {
+	#Basic event information
+	id = WCUTF.2
+	desc = {
+		trigger = {
+			character = 400501
+		}
+		text = EVTDESC_WCUTF.2_loken
+	}
+
+	desc = {
+		trigger = {
+			character = 400502
+		}
+		text = EVTDESC_WCUTF.2_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+
+	immediate = {
+	}
+	option = {
+		name = EVTOPTA_WCUTF.2
+		trigger = {
+			character = 400501
+		}
+		religion = old_gods_worship
+	}
+	option = {
+		name = EVTOPTB_WCUTF.2
+		trigger = {
+			character = 400502
+		}
+		religion = old_gods_worship
+		grant_title_no_opinion  = d_ulduar
+		grant_title_no_opinion  = c_ulduar
+		grant_title_no_opinion  = c_eanorah
+		c_400501 = {
+			death = { death_reason = death_duel killer = PREV }
+		}
+	}
+	after = {
+		any_realm_province = {
+			remove_province_modifier = ulduar_closed_down
+		}
+		set_global_flag = ulduar_opened_flag
+	}
+}
+character_event = {
+	#Basic event information
+	id = WCUTF.3
+	desc = {
+		trigger = {
+			character = 400501
+		}
+		text = EVTDESC_WCUTF.3_loken
+	}
+
+	desc = {
+		trigger = {
+			character = 400502
+		}
+		text = EVTDESC_WCUTF.3_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+
+	immediate = {
+	}
+	option = {
+		name = EVTOPTA_WCUTF.3
+		trigger = {
+			character = 400501
+		}
+		set_player_character = event_target:target_thorim
+		event_target:target_thorim = {
+			grant_title_no_opinion  = d_ulduar
+			grant_title_no_opinion  = c_ulduar
+			grant_title_no_opinion  = c_eanorah
+		}
+		c_400501 = {
+			death = { death_reason = death_suicide }
+		}
+	}
+	option = {
+		name = EVTOPTB_WCUTF.3
+		trigger = {
+			character = 400502
+		}
+		grant_title_no_opinion  = d_ulduar
+		grant_title_no_opinion  = c_ulduar
+		grant_title_no_opinion  = c_eanorah
+		religion = mystery_of_the_makers
+		c_400501 = {
+			death = { death_reason = death_duel killer = PREV }
+		}
+	}
+	after = {
+		event_target:target_thorim = {
+			any_realm_province = {
+				remove_province_modifier = ulduar_closed_down
+				religion = mystery_of_the_makers
+			}
```

Comment:
So, I can vassalize my friend in multiplayer and get religion switch for free.

Reply:
i will make the event default to loken if both are players